homeassistant/components/plugwise/switch.py

```diff
@@ -0,0 +1,86 @@
+"""Plugwise Switch component for HomeAssistant."""
+
+import logging
+
+from Plugwise_Smile.Smile import Smile
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.core import callback
+
+from . import SmileGateway
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(hass, config_entry, async_add_entities):
+    """Set up the Smile switches from a config entry."""
+    api = hass.data[DOMAIN][config_entry.entry_id]["api"]
+    coordinator = hass.data[DOMAIN][config_entry.entry_id]["coordinator"]
+
+    entities = []
+    all_devices = api.get_all_devices()
+    for dev_id, device_properties in all_devices.items():
+        if "plug" in device_properties["types"]:
+            model = "Metered Switch"
+            entities.append(
+                PwSwitch(api, coordinator, device_properties["name"], dev_id, model,)
```

Comment:
```suggestion
                PwSwitch(api, coordinator, device_properties["name"], dev_id, model)
```

Reply:
Weird, normally flake/lint pick up on that